mobile/lib/blocs/map/map_bloc.dart

```diff
@@ -47,8 +47,8 @@ class MapBloc extends Bloc<MapEvent, MapState> {
 
     countries.removeWhere((element) => element.isEmpty);
     countries.sort();
-    airQualityReadings =
-        airQualityReadings.sortByAirQuality(sortCountries: true);
+    airQualityReadings = airQualityReadings
+      ..sortByAirQuality(sortCountries: true);
 
```

Comment:
I am curious about this double dot notation here? 
Isn't it used when one wants to call multiple functions on the same object? I am only seeing one. Please enlighten me here.

Reply:
In this case, we can have one email reserved for automated tests.
https://github.com/airqo-platform/AirQo-frontend/pull/1151#discussion_r1179968304

Int he backend, if the email is the above (`automated-tests@airqo.net`), then we don't send out any email rather return with a success indicating that the email was sent.

@Baalmart

Reply:
We don't need to created this email. we can just have it set in the backend as the "sandbox" email

Reply:
Alright